### ISSUE TITLE
Sum of Array

### ISSUE THREAD

**Issue by billfienberg** [blewis008/vocabulary#1]
I think a function to calculate the sum of all the elements inside an array could be useful here.

**Comment by blewisio** [comment]
@billfienberg, agreed, that would be a nice addition to the library. Feel free to submit a PR, or I'll get to it soon.
